integrated_channels/integrated_channel/exporters/learner_data.py

```diff
@@ -516,39 +557,64 @@ def _collect_certificate_data(self, enterprise_enrollment):
             is_passing: Boolean indicating if the grade is a passing grade or not.
         """
 
-        if self.certificates_api is None:
-            self.certificates_api = CertificatesApiClient(self.user)
-
         course_id = enterprise_enrollment.course_id
         username = enterprise_enrollment.enterprise_customer_user.user.username
+        user_id = enterprise_enrollment.enterprise_customer_user.user_id
 
-        try:
-            certificate = self.certificates_api.get_course_certificate(course_id, username)
-            completed_date = certificate.get('created_date')
-            if completed_date:
-                completed_date = parse_datetime(completed_date)
-            else:
-                completed_date = timezone.now()
+        completed_date = None
+        grade = self.grade_incomplete
+        is_passing = False
+        percent_grade = None
 
-            # For consistency with _collect_grades_data, we only care about Pass/Fail grades. This could change.
-            is_passing = certificate.get('is_passing')
-            percent_grade = certificate.get('grade')
-            grade = self.grade_passing if is_passing else self.grade_failing
+        try:
+            certificate = get_course_certificate(course_id, username)
```

Comment:
Why do we get course certificate by Username here? We should always be getting these items by lms_user_id.